tests/integration/src/test/java/org/apache/pulsar/tests/integration/suites/PulsarSQLTestSuite.java

```diff
@@ -54,20 +51,6 @@ protected PulsarClusterSpec.PulsarClusterSpecBuilder beforeSetupCluster(String c
         return super.beforeSetupCluster(clusterName, specBuilder);
     }
 
-    @Override
-    protected void beforeStartCluster() throws Exception {
-        Map<String, String> envMap = new HashMap<>();
-        envMap.put("managedLedgerMaxEntriesPerLedger", String.valueOf(ENTRIES_PER_LEDGER));
-        envMap.put("managedLedgerMinLedgerRolloverTimeMinutes", "0");
```

Comment:
these settings might be needed for other tests, not just PulsarSQL offloading tests.

Reply:
These configurations are only needed by the test `TestPrestoQueryTieredStorage.java`, they are used to control offload operation. The `TestBasicPresto.java` don't need them.

Reply:
Please take care of that in a separate PR

Reply:
OK, fixed